apps/graphql/src/apps/geoIp/apis/index.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+
+import fetch from '../../../services/fetch/globalFetch';
+
+export type GeoIP = {|
+  isoCountryCode: ?string,
```

Comment:
I think it could be read only with `+`, what do you think?

Reply:
💯
 I always forgot about that 🙈

Reply:
🚀 done